qrisk/stats.py

```diff
@@ -15,13 +15,10 @@
 
 from __future__ import division
 
-from collections import OrderedDict
-from functools import partial
-
 import pandas as pd
 import numpy as np
 import scipy as sp
-import scipy.stats
+import scipy.stats  # noqa
```

Comment:
I would argue that because you were confused by this, other readers will be to. We can just replace it with something like `import scipy.stats as sps` or `from scipy.stats import {}`